src/shogun/lib/observers/ParameterObserverScalar.h

```diff
@@ -52,10 +52,21 @@ namespace shogun
 
 	public:
 		CParameterObserverScalar();
-		CParameterObserverScalar(std::vector<std::string>& parameters);
-		CParameterObserverScalar(
-		    const std::string& filename, std::vector<std::string>& parameters);
-		~CParameterObserverScalar();
+
+        CParameterObserverScalar(std::vector<std::string> &parameters);
```

Comment:
do we even still need this class now?

Reply:
This class is still needed atm since it is used to serialize an observation into the Tensorboard format (which can be read by `tensorboard --logdir lala`).